sdk/keyvault/Azure.Security.KeyVault.Secrets/README.md

```diff
@@ -140,7 +140,7 @@ secret.Tags["foo"] = "updated tag";
 SecretBase updatedSecret = client.Update(secret);
 
 Console.WriteLine(updatedSecret.Name);
-Console.WriteLine(updatedSecret.Value);
+Console.WriteLine(updatedSecret.Vault);
```

Comment:
This will be renamed to `VaultUri` in #7751.

Reply:
Will this change be released in preview4? We suggest to update readme in this way now and update again once this change released.  Or we hold on update until this change release.